scripts/werf-in-image/werf.yaml

```diff
@@ -51,12 +24,17 @@ dockerfile: "./{{ $distro }}.Dockerfile"
     {{- range $_, $channel := $.Channels }}
 ---
 image: "{{ $group }}-{{ $channel }}-{{ $distro }}"
-fromImage: "{{ $distro }}"
-import:
+dockerfile: ./artifact.Dockerfile
+dependencies:
 - image: trdl
-  after: install
-  add: "/usr/local/bin/werf-{{ $group }}-{{ $channel }}"
-  to: /usr/local/bin/werf
+  imports:
+    - type: ImageName
+      targetBuildArg: source_image
+
+args:
+  distro: "{{ $distro }}"
+  source: "/usr/local/bin/werf-{{ $group }}-{{ $channel }}"
+  dest: "/usr/local/bin/werf"
```

Comment:
Let's use the same logic for naming:
- source_image, distro_image
- src, dest or source, destination